Expanded docs of verify_results test helper function (#1488)
OpenSCAP/openscap: docs/contribute/testing.adoc

[source,bash]
----
verify_results TYPE CONTENT_FILE RESULTS_FILE COUNT

verify_results "def" test_probe_foo.xml results.xml 13
----

The function extracts the actual test/definition result, and compares it with the respective test/definition comment.

> So for example, if the test contains `comment="true"`, the test passes only if result of the respective test is `true`,

Reviewer: ```suggestion
For example, if the test contains `comment="true"`, the test passes only if result of the respective test is `true`,
```

if `comment="false"`, a `false` result is expected.

> If the comment is missing or it has other value, it is assumed that the result shouldn't be neither `true` or `false`.

Reviewer: ```suggestion
If the comment is missing or it has other value, it is assumed that the result should be neither `true` nor `false`.
```

NOTE: This function expects that the OVAL `TYPE` is numbered from `1` to `COUNT`
in the `RESULTS_FILE`.

> Type is typically `def` or `tst` for definitions and tests respectively.

Reviewer: ```suggestion
`TYPE` is typically `def` or `tst` for definitions and tests respectively.
```
